pkg/config/config_test.go

```diff
@@ -1295,6 +1295,221 @@ func TestSetConfiguration(t *testing.T) {
 	}
 }
 
+func TestSetLocalConfiguration(t *testing.T) {
+
+	tempConfigFile, err := ioutil.TempFile("", "odoconfig")
```

Comment:
Can you use testingutil/fileutils.go ?

Reply:
I went with uniformity here as all the other tests did this I also went for it